### DISCUSSION TITLE
If a proxy is used, from where will the results of the timings be taken from?

### DISCUSSION THREAD

**Discussion by bushev** [sindresorhus/got#1339] [Q&A] [answered]
#### What would you like to discuss?

Can I use proxy and get timings between the proxy server and target? Or timings will be taken as `got <-> proxy server <-> target`?

#### Checklist

- [x] I have read the documentation.

**Comment by szmarczak** [comment, marked as answer]
Timings always point to the server **you** connect to, in this case it's the proxy server.
